README.md

```diff
@@ -13,8 +13,12 @@ workflows into your own project:
 cd /tmp
 git clone https://github.com/laminas/automatic-releases.git
 cd /path/to/your/project
-cp -r /tmp/automatic-releases/examples/.github ./.github
-git add .github
+install -d .github/workflows
```

Comment:
I'd not change this: `install` is quite obscure, while `cp` is widely understood.

Reply:
Changed to `mkdir -p`, Good thing is that you don't have to type commands from here :)